app/views/govuk_publishing_components/components/_metadata.html.erb

```diff
@@ -18,6 +18,8 @@
   classes << "direction-#{direction}" if local_assigns.include?(:direction)
   classes << "gem-c-metadata--inverse" if inverse
   classes << shared_helper.get_margin_bottom if local_assigns[:margin_bottom]
+
+  ga4_tracking ||= false
```

Comment:
Going through this PR commit by commit so apologies if it comes later, but does this change require a test? Can it be included in this commit?

Reply:
Yep, it has a test :+1: